webmails/snappymail/config/nginx-snappymail.conf

```diff
@@ -0,0 +1,62 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    root /var/www/webmail;
+
+    include /etc/nginx/mime.types;
+
+    # /dev/stdout (Default), <path>, off
+    access_log off;
+
+    # /dev/stderr (Default), <path>, debug, info, notice, warn, error, crit, alert, emerg
+    error_log /dev/stderr notice;
+
+    index index.php;
+
+    # set maximum body size to configured limit
+    client_max_body_size {{ MESSAGE_SIZE_LIMIT|int + 8388608 }};
+
+    location / {
+        try_files $uri $uri/ /index.php$args;
+    }
+
+    location ~ \.php$ {
+        fastcgi_split_path_info ^(.+?\.php)(/.*)$;
+        if (!-f $document_root$fastcgi_script_name) {
+            return 404;
+        }
+        include /etc/nginx/fastcgi_params;
+
+        fastcgi_intercept_errors on;
+        fastcgi_index  index.php;
+
+        fastcgi_keep_conn on;
+
+        fastcgi_pass unix:/var/run/php7-fpm.sock;
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        {% if WEB_WEBMAIL == '/' %}
+        fastcgi_param SCRIPT_NAME $fastcgi_script_name;
+        {% else %}
+        fastcgi_param SCRIPT_NAME {{WEB_WEBMAIL}}/$fastcgi_script_name;
+        {% endif %}
+    }
+
+    location ~ /\.ht {
```

Comment:
I'd rather we denied access to any hidden file here

Reply:
My understanding is that it shouldn't: in your example the dot is a parameter (after the '?').
What would be blocked is http://example.com/webmail/.example

Reply:
I have made the propose change